### DISCUSSION TITLE
Safety information in legacy feed

### DISCUSSION THREAD

**Discussion by NoxArt** [amzn/selling-partner-api-models#4385] [Q&A]
Hi, we've come across an info that the PDF we're linking to in product feed (compliance media) should also be added in image form. We're yet to convert images to JSON (these should use image_locator_ps0x, right?) ... just wanted to confirm, these would also be marked using "PS0x" image type in the XML feed?
Thank you

**Comment by NoxArt** [comment]
Also I uploaded a new picture this way
* feed was processed
* 100% items successful
* I checked the link and it works correctly
* Four hours after the image is still not there on the website
And I don't see any place in administration that would show them ... Edit listing > Images / Manage images only show regular images, not safety/compliance ... and I don't see it in Safety tab either

**Comment by ritikacd** [comment]
Hi @NoxArt,

Currently, legacy Feed types(XML/custom templates) do not support uploading pdf document through product safety image variants, they are only used to upload image files. However, we're pleased to inform you that we are actively working on implementing this functionality, which we expect to be available soon. 

 In the meantime, we offer two alternative methods for uploading compliance documents: you can either utilize our new Listings APIs or access the Seller Central account health page. 

For more detailed information on updating GPSR compliance documents using the Listings Items API and JSON Listings Feeds, we encourage you to refer to our recent [announcement](https://developer-docs.amazon.com/sp-api/changelog/developers-can-use-attributes-in-their-programmatic-listings-submissions-to-comply-with-gpsr), which provides comprehensive guidance on these processes." 

For Seller Central feature, please refer to this [announcement](https://developer-docs.amazon.com/sp-api/changelog/sellers-can-prove-their-gpsr-compliance-on-their-account-health-page). 

I hope this helps!

Thank you, 
Ritika
Solutions Architect
Selling Partner API Services
